posthog/ai/openai/openai_async.py

```diff
@@ -394,8 +452,6 @@ async def create(
             posthog_distinct_id: Optional ID to associate with the usage event.
             posthog_trace_id: Optional trace UUID for linking events.
             posthog_properties: Optional dictionary of extra properties to include in the event.
-            posthog_privacy_mode: Whether to store input and output in PostHog.
```

Comment:
Are we removing args or just docs were wrong? If the former, we'd need a major release. I'll assume it's the latter

Reply:
this was incorrect docs. fixing